.github/ISSUE_TEMPLATE/generic_issue.yaml

```diff
@@ -0,0 +1,126 @@
+name: Report a generic issue
+description: Report a generic issue related to Home Assistant Intents.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue form is for reporting issues related to Home Assistant Intents
+
+        If you have a feature or enhancement request, please use the [feature request][fr] section of our [Community Forum][fr].
+
+        [fr]: https://community.home-assistant.io/c/feature-requests
+  - type: dropdown
+    validations:
+      required: false
+    attributes:
+      label: Is the issue related to a specific language?
+      description: >
+        Select the language to which this issue is related.
+      options:
+        - ar
+        - bg
+        - bn
+        - ca
+        - cs
+        - da
+        - de
+        - de-CH
+        - el
+        - en
+        - es
+        - fa
+        - fi
+        - gl
+        - he
+        - hi
+        - hr
+        - hu
+        - id
+        - is
+        - it
+        - ka
+        - kn
+        - lb
+        - lt
+        - lv
+        - ml
+        - mn
+        - ms
+        - nb
+        - nl
+        - pl
+        - ro
+        - ru
+        - sk
+        - sl
+        - sr
+        - sv
+        - sw
+        - te
+        - tr
+        - uk
+        - ur
+        - vi
+        - zh-cn
+        - zh-hk
+        - zh-tw
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: The problem
+      description: >-
+        Describe the issue you are experiencing here, to communicate to the
+        maintainers. Tell us what you were trying to do and what happened.
+
+        Provide a clear and concise description of what the problem is.
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+  - type: input
+    id: version
+    validations:
+      required: true
+    attributes:
+      label: What version of Home Assistant Core has the issue?
+      placeholder: core-
+      description: >
+        Can be found in: [Settings -> About](https://my.home-assistant.io/redirect/info/).
+
+        [![Open your Home Assistant instance and show your Home Assistant version information.](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/)
+  - type: input
+    attributes:
+      label: What was the last working version of Home Assistant Core?
+      placeholder: core-
+      description: >
+        If known, otherwise leave blank.
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: What type of installation are you running?
```

Comment:
This is not necessary.

Also, we need to be careful, the more we ask, the more likely people don't finish the form.